src/aurora.ts

```diff
@@ -124,6 +124,11 @@ export interface AuroraProps {
    * @default false
    */
   readonly skipAddRotationMultiUser?: boolean;
+  /**
+   * Days between password rotations for the users.
+   * @default 45
```

Comment:
Why are we overriding the `aws-cdk-lib` default here? 
```suggestion
   * @default - fallthrough to aws-cdk-lib, currently 30 days
```

Reply:
If you want to apply a ClickUp opinion, that should happen on an internal library.